src/main/java/io/reactivex/Flowable.java

```diff
@@ -6740,12 +6744,16 @@ public final void blockingSubscribe(Subscriber<? super T> subscriber) {
     }
 
     /**
-     * Collects items emitted by the source Publisher into a single mutable data structure and returns
+     * Collects items emitted by the finite source Publisher into a single mutable data structure and returns
      * a Single that emits this structure.
      * <p>
      * <img width="640" height="330" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/collect.png" alt="">
      * <p>
      * This is a simplified version of {@code reduce} that does not need to return the state on each pass.
+     * <p>
+     * Note that this operator requires the upstream to signal {@code onComplete} for the accumulator to
+     * be emitted. Sources that are infinite and never complete will never emit anything through this
```

Comment:
"accumulator to emit" or "accumulation to be emitted" or "accumulated object to be emitted".  There are other occurrences of this in the PR.